pages/deposit/index.vue

```diff
@@ -0,0 +1,304 @@
+          autosize
+          @keyup.enter="bindDepositAllToGodwoken"
+        >
+        </el-input>
+      </el-form-item>
+      <div class="balance">{{ balance }} {{ asset.symbol }}</div>
+      <el-form-item :label="t_('Money')" prop="amount"></el-form-item>
+      <div v-loading="feeLoading" class="balance fee">{{ fee }} CKB</div>
+      <el-form-item :label="t_('Fee')"></el-form-item>
+    </el-form>
+    <el-button
+      type="primary"
+      :loading="loading"
+      :disabled="form.address === ''"
+      class="deposit"
+      @click="bindDepositAllToGodwoken"
+    >
+      {{ t_('Deposit') }}
+    </el-button>
+  </div>
+</template>
+<script>
+import {
+  Address,
+  AddressType,
+  AmountUnit,
+  Builder,
+} from '@lay2/pw-core'
+import { buildDepositSudtSignMessage, minCkbToDeposit } from '~/assets/js/sudt/deposit-sudt-builder'
+import { generateDepositLock } from '~/assets/js/sudt/deposit-lock-generator'
+import { getSUDTSignCallback } from '~/assets/js/sudt/sudt-tranfer'
+import { isCkbEnough } from '~/assets/js/sudt/utils'
+
+export default {
+  data() {
+    const name = this.$route.query.name
+    const checkAddress = (_rule, value, callback) => {
+      try {
+        if (/^0x[a-fA-F0-9]{40}$/.test(value)) {
+          // eslint-disable-next-line no-new
+          new Address(value, AddressType.eth)
+          callback()
+        } else {
+          callback(new Error(this.t_('BadAddress')))
+        }
+      } catch (error) {
+        callback(new Error(this.t_('BadAddress')))
+      }
+    }
+    return {
+      rules: {
+        address: [
+          {
+            required: true,
+            message: this.t_('PleaseAddress'),
+            trigger: 'change',
+          },
+          { validator: checkAddress, trigger: 'change' },
+        ],
+      },
+      feeLoading: false,
+      loading: false,
+      form: {
+        address: '',
+      },
+      fee: '0.00002040',
+      feeRate: 1000,
+      name,
+    }
+  },
+  computed: {
+    provider() {
+      return this.$store.state.provider
+    },
+    asset() {
+      const assets = this.$store.state.assets
+      for (const asset of assets) {
+        if (asset.symbol === this.name) {
+          return asset
+        }
+      }
+      return {}
+    },
+    decimals() {
+      return this.asset.decimals || AmountUnit.shannon
+    },
+    balance() {
+      const asset = this.asset
+      if (asset.symbol) {
+        const balance = asset.sudt ? asset.sudtAmount : asset.capacity
+        return balance.toString(this.decimals, {
+          commify: true,
+          fixed: this.decimals >= 4 ? 4 : this.decimals || undefined,
+        })
+      }
+      return ''
+    },
+    sudtAmount() {
+      return this.asset.sudtAmount
+    },
+    sudtTokenId() {
+      if (this.asset && this.asset.typeScript) {
+        return this.asset.typeScript.args
+      }
+      return ''
+    },
+    isTestNet() {
+      return process.env.NODE_ENV === 'development'
```

Comment:
`NODE_ENV === 'development'' is usually used in development environments, this ENV tends to introduce some debug features causing additional performance overhead, we may also use `TESTNET` in production environments, so we should add an environment variable in .env to distinguish network instead of `NODE _EN`

```env
# NETWORK_TYPE=MAINNET
# NETWORK_TYPE=TESTNET
```

Reply:
> `NODE_ENV === 'development'' is usually used in development environments, this ENV tends to introduce some debug features causing additional performance overhead, we may also use `TESTNET`in production environments, so we should add an environment variable in .env to distinguish network instead of`NODE _EN`
> 
> ```shell
> # NETWORK_TYPE=MAINNET
> # NETWORK_TYPE=TESTNET
> ```

Yes, it's better.